simp_le.py

```diff
@@ -990,8 +998,8 @@ def create_parser():
     )
     reg.add_argument(
         '--tos_sha256', help='SHA-256 hash of the contents of Terms Of '
-        'Service URI contents.', default='6373439b9f29d67a5cd4d18cbc7f2'
-        '64809342dbf21cb2ba2fc7588df987a6221', metavar='HASH',
+        'Service URI contents.', default=get_le_tos_hash(LE_PRODUCTION_URI),
```

Comment:
Should we make the default `None` and make the http request later, once the CLI as fully started? This will make it possible to display some feedback to the user while we send stuff over the internet which can take (and "block" the CLI for) a few seconds.

---- 

Thanks for the patch otherwise!